docs/src/plots/TerminalVelocityComparisons.jl

```diff
@@ -218,23 +232,39 @@ Ch_snow = [snow_terminal_velocity_individual_Chen(snow, Chen2022.snow_ice, ρ_ai
 
 # single drop comparison
-p1 = PL.plot(D_r_range *1e3,  SB_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label  = "Rain-SB2006", color = :blue)
-p1 = PL.plot!(D_r_range * 1e3, Ch_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022", color = :red)
-p1 = PL.plot!(D_r_range * 1e3, M1_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label = "Rain-1M", color = :green)
+p1 = PL.plot(D_r_range *1e3,  SB_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label  = "Rain-SB2006", color = 1)
+p1 = PL.plot!(D_r_range * 1e3, Ch_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022", color = 2)
+p1 = PL.plot!(D_r_range * 1e3, M1_rain, linewidth = 3, xlabel = "D [mm]", ylabel = "terminal velocity [m/s]", label = "Rain-1M", color = 3)
+p1 = PL.scatter!(D_Gunn_Kinzer * 1e3, u_Gunn_Kinzer, ms = 3, color = 4, label = "Gunn and Kinzer (1949)")
 # group velocity comparison
-p2 = PL.plot(q_rain_range * 1e3,  SB_rain_bN, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-SB2006 [ND]", linestyle = :dot, color = :blue)
-p2 = PL.plot!(q_rain_range * 1e3, Ch_rain_bN, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022 [ND]", linestyle = :dot, color = :red)
-p2 = PL.plot!(q_rain_range * 1e3, SB_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-SB2006 [M]", color = :blue)
-p2 = PL.plot!(q_rain_range * 1e3, Ch_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022 [M]", color = :red)
-p2 = PL.plot!(q_rain_range * 1e3, M1_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-default-1M [M]", color = :green)
+p2 = PL.plot(q_rain_range * 1e3,  SB_rain_bN, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-SB2006 [ND]", linestyle = :dot, color = 1)
+p2 = PL.plot!(q_rain_range * 1e3, Ch_rain_bN, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022 [ND]", linestyle = :dot, color = 2)
+p2 = PL.plot!(q_rain_range * 1e3, SB_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-SB2006 [M]", color = 1)
+p2 = PL.plot!(q_rain_range * 1e3, Ch_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-Chen2022 [M]", color = 2)
+p2 = PL.plot!(q_rain_range * 1e3, M1_rain_bM, linewidth = 3, xlabel = "q_rain [g/kg]", ylabel = "terminal velocity [m/s]", label = "Rain-default-1M [M]", color = 3)
+# SB2006 group velocities vs. mean radius
+p3 = PL.scatter(r_mean, SB_rain_bN_rm, ms = 1.0, markerstrokewidth=0, xlabel = "r_mean [μm]", ylabel = "terminal velocity [m/s]", label = "Rain-SB2006 [ND]")
```

Comment:
I don't understand why the bottom two panels `Rain-SB2006` (blue color) is a cloud of points and the other two are lines?

Reply:
For SB2006 with limiters, terminal velocity is a function of both q and N while for the others it is only a function of mean radius. So in the plot we see multiple terminal velocity values for one value of mean radius for the equation with limiters while there is only one value for other cases.

Reply:
This is interesting! Thank you, I guess I wasn't aware of that. All that scatter is only present for the small drops, right? After we get out of the region where the limiters are important, the two options will be quite similar right?

If yes, could we either change the x axis on the bottom panels to extend further till about 2 mm? Or highlight in the description that we are zooming into the region of interest and that outside of that they are the same?

Thank you for catching and fixing all of this!

Reply:
The limiters are important for large droplets too. For medium size droplets when both N and lambda are in the defined range then the two velocities are similar. I extended x axis to 2 mm and now the effect of limiters is more clear on the figure.